### PR TITLE
change: more compact mention view

### PR DESCRIPTION
**Pull request by singularity-s0**

### CODE REVIEW

File: DanXiUI/Forum/Views/MentionView.swift [hunk 1]
```diff
@@ -31,34 +31,28 @@ struct MentionView: View {
...
-                Text(verbatim: "##\(String(id))")
-                Spacer()
-                Text(timeUpdated.formatted(.relative(presentation: .named, unitsStyle: .wide)))
-            }
-            .font(.caption)
-            .foregroundColor(.secondary)
-            .padding(.top, 1.0)
         }
         .padding(.horizontal, 12.0)
         .padding(.vertical, 7.0)
```
Comment (ivanfei-1) [inline DanXiUI/Forum/Views/MentionView.swift:58]:
```
.padding(.top, 8.0)
.padding(.bottom, 12.0)
```

Comment (singularity-s0) [reply DanXiUI/Forum/Views/MentionView.swift:58]: You can commit to this branch yourself

**Comment by singularity-s0** [comment]
> I don't like this.

Why?

**Comment by singularity-s0** [comment]
The goal is to make it thin.

**Review by ivanfei-1 — APPROVED** [review]

**Pull request merged**